stacks/camunda-ci-postgresql-15.0/docker-stack.yml

```diff
@@ -0,0 +1,11 @@
+---
+version: '3'
+services:
+  main:
+    deploy:
+      restart_policy:
+        condition: on-failure
+    image: gcr.io/ci-30-162810/postgresql:15.0v0.1.0
```

Comment:
it was `15.0v0.2.0` in https://github.com/camunda/infra-core/pull/4683/files but 0.1.0 here, a typo?

Reply:
It should be 0.1.0 everywhere since we are starting fresh with this, will fix it in the other PR